system-tests/projects/config-cjs-and-esm/ts-cjs-with-invalid-esm-only-import/yarn.lock

```diff
@@ -0,0 +1,47 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Not related to this PR but somehow this project made it into develop w/o a yarn.lock, so I added one to fix the CI